runtime/integration-tests/src/pools/utils/env.rs

```diff
@@ -338,10 +339,17 @@ pub struct TestEnv {
 	pub events: Arc<Mutex<EventsStorage>>,
 }
 
+pub type Block = sp_runtime::generic::Block<Header, UncheckedExtrinsic>;
+pub type UncheckedExtrinsic =
+	sp_runtime::generic::UncheckedExtrinsic<u32, <Runtime as frame_system::Config>::Call, u64, ()>;
+
```

Comment:
Please import our Block type from `runtime_common`. Please use the `centrifuge::UncheckedExtrinsic` instead of generic one.